src/services/expenses.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
+
+let expenses = [];
+
+const getInitial = () => {
+  expenses = [];
+};
+
+const getAll = ({
+  userId,
+  categories,
+  from,
+  to,
+}) => {
+  let filteredExpenses = [...expenses];
+
+  const category = Array.isArray(categories)
+    ? categories
+    : [categories];
+
+  if (userId) {
+    const userIdToNumber = +userId;
+
+    filteredExpenses = filteredExpenses
+      .filter(exp => exp.userId === userIdToNumber);
+  }
```

Comment:
I created separate variable outside , where change type of variable, because i did it only once, if I did it inside itterable function , I'll did on each itterable